Chapters/6_Qualified_identifiers.md

```diff
@@ -18,13 +18,29 @@ The following *context qualifiers* are available:
 A "line" in the context of a file content refers to a sequence of characters that ends with a line break. This line can contain text, code, or any other form of data. In this specification, the line break is the ASCII LF character.
 The "lines" qualifier allows to designate a line range inside a content.
 The range can be a single line number, or a pair of line numbers separated by the ASCII `-` character.
+Line numbers start from 1.
 
 For example, [`swh:1:cnt:4d99d2d18326621ccdd70f5ea66c2e2ac236ad8b;lines=9-15`](https://archive.softwareheritage.org/swh:1:cnt:4d99d2d18326621ccdd70f5ea66c2e2ac236ad8b;lines=9-15)
 designates the function `generate_intput_stream` that is found at lines 9 to 15 of the *content* with core SWHID `swh:1:cnt:4d99d2d18326621ccdd70f5ea66c2e2ac236ad8b`.
 
 Notice that the notion of "line number" is not always meaningful: the content
 may be a binary file, or a file that uses non standard line termination character(s).
 
+### 6.1.2 Bytes qualifier
+
+To overcome the limitations of the lines qualifier, the bytes qualifier allows
+to designate a byte range inside a content.  The range can be a single byte number, or a pair of byte numbers separated by `-`.
+Byte numbers start from 0.
```

Comment:
again, make it explicit that range is inclusive at both ends.

Reply:
Done in b82a54b28e81dcd45a4590fba19cac33c2025346